odoo/models.py

```diff
@@ -277,17 +277,18 @@ def _field_create(self):
         params = {
             'model': self._name,
             'name': self._description,
+            'is_business': self._is_business,
             'info': next(cls.__doc__ for cls in type(self).mro() if cls.__doc__),
             'state': 'manual' if self._custom else 'base',
             'transient': self._transient,
         }
         cr.execute(""" UPDATE ir_model
-                       SET name=%(name)s, info=%(info)s, transient=%(transient)s
+                       SET name=%(name)s, is_business=%(is_business)s, info=%(info)s, transient=%(transient)s
                        WHERE model=%(model)s
                        RETURNING id """, params)
         if not cr.rowcount:
-            cr.execute(""" INSERT INTO ir_model (model, name, info, state, transient)
-                           VALUES (%(model)s, %(name)s, %(info)s, %(state)s, %(transient)s)
+            cr.execute(""" INSERT INTO ir_model (model, name, is_business, info, state, transient)
+                           VALUES (%(model)s, %(name)s, %(is_business)s, %(info)s, %(state)s, %(transient)s)
```

Comment:
Wait, don't merge too quick! I have another PR (#15070) that refactors this code!